config/default/kustomization.yaml

```diff
@@ -70,3 +70,38 @@ vars:
 #    kind: Service
 #    version: v1
 #    name: webhook-service
+
+# since we add a prefix to resource names, we need to add in those new names of
+# those resources into the resources that reference them
```

Comment:
For my info, where do we add that prefix?

Reply:
It gets added by the `namePrefix` field in [`config/default/kustomization.yaml`](https://github.com/konflux-ci/build-service/blob/main/config/default/kustomization.yaml#L9)